composer.json

```diff
@@ -1,62 +1,70 @@
 {
-	"name" : "oat-sa/lib-flysystem-filecache",
-	"authors" : [{
-			"name" : "Open Assessment Technologies S.A.",
-			"homepage" : "http://www.taotesting.com"
-		}, {
-			"name" : "Jérôme Bogaerts",
-			"role" : "Developer"
-		}, {
-			"name" : "Joel Bout",
-			"role" : "Developer"
-		}, {
-			"name" : "Bertrand Chevrier",
-			"role" : "Developer"
-		}, {
-			"name" : "Lionel Lecaque",
-			"role" : "Developer"
-		}, {
-			"name" : "Patrick Plichart",
-			"role" : "Developer"
-		}, {
-			"name" : "Dieter Raber",
-			"role" : "Developer"
-		}, {
-			"name" : "Somsack Sipasseuth",
-			"role" : "Developer"
-		}, {
-			"name" : "Christophe Garcia",
-			"role" : "Developer"
-		}
-	],
-	"description" : "flysystem cache Adapter",
-	"support" : {
-		"forum" : "http://forum.taotesting.com",
-		"issues" : "http://forge.taotesting.com"
-	},
-	"license" : "GPL-2.0-only",
-	"keywords" : [
-		"flysystem",
-		"oat",
-		"flysystem adapter",
-                "cache"
-	],
-	"type" : "library",
-	"require" : {
-		"php" : ">=5.5",
-		"league/flysystem" : "^1.0.25"
-	    },
-            
-        "require-dev": {
-            "phpunit/phpunit": "~4.4@dev",
-            "sebastian/global-state": "~1.0@dev",
-            "mikey179/vfsStream": "1.4.0"
-        },
-	"minimum-stability" : "dev",
-	"autoload" : {
-		"psr-4" : {
-			"oat\\flysystem\\Adapter\\" : "src",
-			"oat\\libFlysystemFilecache\\test\\" : "test"
-		}
-	}
+  "name": "oat-sa/lib-flysystem-filecache",
+  "authors": [
+    {
+      "name": "Open Assessment Technologies S.A.",
+      "homepage": "http://www.taotesting.com"
+    },
+    {
+      "name": "Jérôme Bogaerts",
+      "role": "Developer"
+    },
+    {
+      "name": "Joel Bout",
+      "role": "Developer"
+    },
+    {
+      "name": "Bertrand Chevrier",
+      "role": "Developer"
+    },
+    {
+      "name": "Lionel Lecaque",
+      "role": "Developer"
+    },
+    {
+      "name": "Patrick Plichart",
+      "role": "Developer"
+    },
+    {
+      "name": "Dieter Raber",
+      "role": "Developer"
+    },
+    {
+      "name": "Somsack Sipasseuth",
+      "role": "Developer"
+    },
+    {
+      "name": "Christophe Garcia",
+      "role": "Developer"
+    }
+  ],
+  "description": "flysystem cache Adapter",
+  "support": {
+    "forum": "http://forum.taotesting.com",
+    "issues": "http://forge.taotesting.com"
+  },
+  "license": "GPL-2.0-only",
+  "keywords": [
+    "flysystem",
+    "oat",
+    "flysystem adapter",
+    "cache"
+  ],
+  "type": "library",
+  "require": {
+    "php": ">=7.4",
```

Comment:
league/flysystem ^2 supports PHP 7.2. So, I guess we can be a tad more generous here :)